com.github.CineEncoder.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.CineEncoder
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: CineEncoder
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
```

Comment:
What does it need host access for ?

Reply:
This is for accessing disks, according to the purpose of the program, for transcoding video files.